src/search.rs

```diff
@@ -654,9 +693,15 @@ mod tests {
             .await?;
         let t2 = index.set_sortable_attributes(["title"]).await?;
 
-        t2.wait_for_completion(client, None, None).await?;
-        t1.wait_for_completion(client, None, None).await?;
-        t0.wait_for_completion(client, None, None).await?;
+        // the vector search has longer indexing times leading to the timeout being triggered
+        let timeout = if cfg!(feature = "experimental-vector-search") {
+            Some(Duration::from_secs(120))
```

Comment:
I did a few tests, and on my side it never took 120s, the quickest execution took 150s but most of the time I was over 200s